src/Posts.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment,useState } from "react";
+
+const posts = [
```

Comment:
Move this to another file and call it `postsData.js` and import it here.

Reply:
Done.